cluster/manifests/kube-dns/svc-kube-dns-backup.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: kube-dns-backup
+  namespace: kube-system
+  labels:
+    application: kube-dns-backup
+    kubernetes.io/cluster-service: "true"
+    kubernetes.io/name: "KubeDNS"
+spec:
+  selector:
+    application: kube-dns-backup
+  clusterIP: 10.3.0.11
```

Comment:
This could potentially conflict with other services in existing clusters. I don't see a good way to roll this out other than fix it one at a time in case there are conflicts.

Reply:
@mikkeloscar I've extended the cluster IPs to /16 in the previous commit. Since they seem to be allocated randomly, the chances of getting a conflict are rather low.